### ISSUE TITLE
Grab and Release Alerts

### ISSUE THREAD

**Issue by terracoda** [phetsims/friction#130]
In issue #126 , we are implementing "grab buttons".

These buttons need alerts. Alerts have been added to the design document in section
[Grab button alerts for Grabbed & Released States](https://docs.google.com/document/d/1v7GM_dNZs86jcRn2X03D2t7HFDb4dQ1h4ayDGXTu63U/edit#heading=h.5pe7t0653eze)

Copying the alerts here for simplicity.
**On Grab:**
1. Initial grab when books are not touching (page load and after a reset):
- Grabbed. Chemistry book rests lightly on Physics book. Use W, A, S, or D keys to move book. Space to release.
2. Grab when books are not touching:
- Grabbed. Lightly on Physics book. Move down to rub harder.
3. Grab when books (books touching):
- Grabbed. On Physics book. Move fast or slow.

**On Release: (keeping it simple)**
- Released. (and then the button’s label will be read automatically when focus returns to the button.)

**Note on an alternative:** 
Considering the new similar interaction hints being implemented in #124 and #125, the grab alerts might need to be reduced to:
1. Initial grab when books are not touching (page load and after a reset):
- Grabbed. Chemistry book rests lightly on Physics book. Use W, A, S, or D keys to move book. Space to release.
2. Grab when books are not touching:
- Grabbed. Lightly on Physics book.
3. Grab when books (books touching):
- Grabbed. On Physics book.

**Comment by terracoda** [comment]
@zepumph, in my brief retrospective upon posting this issue, I think it might be best to start the grab alerts with the shorter suggested alternative suggestion.

**Comment by zepumph** [comment]
I added grab/release alerts to the branch in the above commits. @terracoda here is the latest version of grab button implementation. This has everything from #126 as well as these alerts, please review.

https://phet-dev.colorado.edu/html/friction/1.5.0-grabButton.2/phet/friction_all_phet.html

Tagging @emily-phet and @jessegreenberg in case they want to play with it too.

**Comment by terracoda** [comment]
@zepumph, this works great! Awesome. I made a mistake in the design, though.
I thought I was trying to avoid directly saying that the Chemistry book is "on the Physics book" because technically it is always on the Physics book.

Please change the on-Physics-book grab alert to "Grabbed. Rub fast or slow."

Apologies for the lack of forethought on my part.

I've updated the design document, too, striking out old alert.

**Comment by zepumph** [comment]
Implemented above. Please note that number (2) alert from above is still "Grabbed. Lightly on Physics book."

Back to you to review, comment, or close.

**Comment by terracoda** [comment]
Thanks @zepumph, these alerts are working correctly.
Closing.
